OracleWebCenterContent/README.md

```diff
@@ -0,0 +1,16 @@
+# Oracle WebCenter Content Container Image
+
+This project offers scripts to build an Oracle WebCenter Content container image based on 12.2.1.4.0 release. Use this documentation to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Content, see the [Oracle WebCenter Content 12.2.1.4.0 Online Documentation](https://docs.oracle.com/en/middleware/webcenter/content/12.2.1.4/index.html).
+
+## For Building & Running the WebCenter Content Container images and Using
+
+**IMPORTANT**
+- Refer [Oracle WebCenter Content 12.2.1.4.0](dockerfiles/README.md) for detailed set up.
+
+## License
+To download and run the WebCenter Content, regardless whether inside or outside a container, and regardless of the distribution, you must download the binaries from the Oracle content and accept the license indicated on that page.
```

Comment:
The correct name is the Oracle Software Delivery Cloud, not "the Oracle content" unless your products come from somewhere different from all the rest.